pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -375,6 +383,18 @@ func CreateGCECloud(config *CloudConfig) (*GCECloud, error) {
 
 	operationPollRateLimiter := flowcontrol.NewTokenBucketRateLimiter(10, 100) // 10 qps, 100 bucket size.
 
+	enableAlphaApiMap := make(map[string]bool)
+	for _, apiName := range config.EnableAlphaAPI {
+		if _, ok := SupportedAlphaApi[apiName]; !ok {
+			return nil, fmt.Errorf("Alpha API %q is not supported.", apiName)
+		}
+		if strings.ToLower(apiName) == "all" {
```

Comment:
Shouldn't this check be done first (before looks up in `SupportedAlphaApi`)? Otherwise if `apiName` is `all`, since that string won't exist in `SupportedAlphaApi`, it will error out before it gets here?